docs/firebase.md

```diff
@@ -400,30 +400,49 @@ cd $PROJECT_ROOT/functions
 npx genkit start -- npx tsx --watch src/index.ts
 ```
 
+or
+  
+```posix-terminal
+cd $PROJECT_ROOT/functions
+
+npm run genkit:start
+```
+
+You can now navigate to the URL printed by the `genkit start` command to access.
+
 ## Optional: Developing using Firebase Local Emulator Suite
 
 Firebase offers a
 [suite of emulators for local development](/docs/emulator-suite), which you can
 use with Genkit.
 
+First, You will need to launch the Genkit Dev UI:
+
+```posix-terminal
+cd $PROJECT_ROOT/functions
+
+npx genkit start -- npx tsx --watch src/index.ts
+```
+
+or
+  
+```posix-terminal
+cd $PROJECT_ROOT/functions
+
+npm run genkit:start
+```
+
 To use Genkit with the Firebase Emulator Suite, start the Firebase emulators
 like this:
 
 ```posix-terminal
-GENKIT_ENV=dev firebase emulators:start --inspect-functions
+firebase emulators:start --inspect-functions
```

Comment:
The intent here is that we want to show how to "attach" the dev ui to the code running inside the emulator. With 0.9 it's actually even easier:

```posix-terminal
npx genkit start -- firebase emulators:start --inspect-functions
```

Reply:
just tested to make sure it works -- the command must be run from the `functions` folder

Reply:
Sure! I just wanted to add the npm command in the new template to avoid some friction just in case people do not want to use `npx`. I also wanted to share how to run the firebase emulator with the genkit UI just in case someone wants to use it because it is very useful when you developing firebase apps.

Reply:
`npx genkit start -- firebase emulators:start --inspect-functions` is how you run emulator with dev ui attached. The way you have written makes to dev ui connect to a separate instance of the code, so dev ui is not connected to the emulator.

Reply:
let me modify the docs to add that option, I do not think that is in the docs

Reply:
@pavelgj what about now?
https://github.com/firebase/genkit/pull/1305/commits/93feb2947a7c3cad8924e750d12086d986585562